locales/eo.json

```diff
@@ -598,5 +598,43 @@
     "diagnosis_basesystem_hardware_board": "Servilo-tabulo-modelo estas {model}",
     "diagnosis_description_web": "Reta",
     "domain_cannot_add_xmpp_upload": "Vi ne povas aldoni domajnojn per 'xmpp-upload'. Ĉi tiu speco de nomo estas rezervita por la XMPP-alŝuta funkcio integrita en YunoHost.",
-    "group_already_exist_on_system_but_removing_it": "Grupo {group} jam ekzistas en la sistemaj grupoj, sed YunoHost forigos ĝin …"
+    "group_already_exist_on_system_but_removing_it": "Grupo {group} jam ekzistas en la sistemaj grupoj, sed YunoHost forigos ĝin …",
+    "diagnosis_mail_outgoing_port_25_blocked_relay_vpn": "Iuj provizantoj ne lasos vin malŝlosi elirantan havenon 25 ĉar ili ne zorgas pri Neta Neŭtraleco. <br> - Iuj el ili provizas la alternativon de <a href='https://yunohost.org/#/smtp_relay'> uzante retpoŝtan servilon </a> kvankam ĝi implicas, ke la relajso povos spioni vian retpoŝtan trafikon. <br> - Amika privateco estas uzi VPN * kun dediĉita publika IP * por pretervidi ĉi tiun specon. de limoj. Vidu <a href='https://yunohost.org/#/vpn_avantage'> https://yunohost.org/#/vpn_avantage </a> <br> - Vi ankaŭ povas konsideri ŝanĝi al <a href = 'https : //yunohost.org/#/isp '> pli neta neŭtraleco-amika provizanto </a>",
+    "diagnosis_mail_fcrdns_nok_details": "Vi unue provu agordi la inversan DNS kun <code>{ehlo_domain}</code> en via interreta enkursigilo aŭ en via retprovizanta interfaco. (Iuj gastigantaj provizantoj eble postulas, ke vi sendu al ili subtenan bileton por ĉi tio).",
+    "diagnosis_mail_fcrdns_nok_alternatives_4": "Iuj provizantoj ne lasos vin agordi vian inversan DNS (aŭ ilia funkcio povus esti rompita ...). Se vi spertas problemojn pro tio, konsideru jenajn solvojn: <br> - Iuj ISP provizas la alternativon de <a href='https://yunohost.org/#/smtp_relay'> uzante retpoŝtan servilon </ a > kvankam ĝi implicas, ke la relajso povos spioni vian retpoŝtan trafikon. <br> - Interreta privateco estas uzi VPN * kun dediĉita publika IP * por preterpasi ĉi tiajn limojn. Vidu <a href='https://yunohost.org/#/vpn_avantage'> https://yunohost.org/#/vpn_avantage </a> <br> - Finfine eblas ankaŭ <a href = 'https : //yunohost.org/#/isp '> ŝanĝo de provizanto </a>",
+    "diagnosis_display_tip": "Por vidi la trovitajn problemojn, vi povas iri al la sekcio pri Diagnozo de la reteja administrado, aŭ funkcii \"yunohost diagnosis show --issues\" el la komandlinio.",
+    "diagnosis_ip_global": "Tutmonda IP: <code> {global} </code>",
+    "diagnosis_ip_local": "Loka IP: <code> {local} </code>",
```

Comment:
Many spaces in between those tags that shouldnt be there ... not really a huge deal but still will change the way it's displayed :/